tests/test_extended_converters.py

```diff
@@ -46,6 +46,7 @@ def with_integer_to_float(sample, expect):
             number: 5
             """,
         )
+        sample.datafile._last_load = 0.0
```

Comment:
Given this has to be added after every `write()` where the class is preconstructed, it makes me think that maybe that shouldn't be in utils.py, but should instead be a method on the datafile itself, so that setting `_last_load` can just be embedded there?

Reply:
These calls to `write()` are meant to emulate external file changes that `datafiles` is agnostic about. The `Mapper` object should detect the changed file automatically without needing to manipulate its internal state.